rpc/rpc-wrappers.go

```diff
@@ -664,21 +664,21 @@ func (vac ValidationAuthorityClient) PerformValidation(ctx context.Context, doma
 
 // IsSafeDomain returns true if the domain given is determined to be safe by an
 // third-party safe browsing API.
-func (vac ValidationAuthorityClient) IsSafeDomain(ctx context.Context, req *core.IsSafeDomainRequest) (*core.IsSafeDomainResponse, error) {
-	data, err := json.Marshal(req)
+func (vac ValidationAuthorityClient) IsSafeDomain(ctx context.Context, domain string) (isSafe bool, err error) {
```

Comment:
Changing the arguments of a RPC method must be config gated or implemented as another method as this will fail if passed a message using the old format during/after a deploy.